src/aiortc/contrib/media.py

```diff
@@ -149,7 +149,7 @@ def player_worker(
                         break
             elif isinstance(frame, VideoFrame) and video_track:
                 if frame.pts is None:  # pragma: no cover
-                    logger.warning(
+                    logger.debug(
```

Comment:
why?

Reply:
It was logging every frame. Not sure why. Possibly it could be a clue as to why Chrome doesn't work

Reply:
In this case `packet` is skipping. Please, share with me your test (in server side).

Reply:
It is better to keep this log as warning (because it's a problem to skip packets or frames).

Reply:
You're right, I'll put it back. And I don't seem to trigger this log anymore. Not sure what changed.

> Please, share with me your test (in server side)

I'm simply running `python webcam.py --no-transcode` from this branch on a Pi with a Logitech c920 attached.